lib/collection/variable-list.js

```diff
@@ -38,7 +39,17 @@ _.inherit((
             var proxies = this._environments;
             environments.reduce(function (parent, child) {
                 return (proxies.push(VariableList.proxy(parent, VariableList.objectify(child))), _.last(proxies));
-            }, {}); // @todo: this blank object can have global default
+            }, {
```

Comment:
add a todo that these needs to be able to have external definitions. else it is super hard to find where they are defined.